server/src/main/java/org/elasticsearch/search/query/PartialHitCountCollector.java

```diff
@@ -81,43 +80,4 @@ boolean isThresholdReached() {
             return numCollected.getAcquire() >= totalHitsThreshold;
         }
     }
-
-    static class CollectorManager implements org.apache.lucene.search.CollectorManager<PartialHitCountCollector, Void> {
```

Comment:
This functionality is now folded into QueryPhaseCollectorManager.EmptyHits. This small collector manager has been moved to PartialHitCountCollectorTests to allow for testing this collector in concurrent scenarios (as opposed to testing the collector only in sequential collection scenarios).